x-pack/plugins/graph/public/plugin.ts

```diff
@@ -104,7 +104,7 @@ export class GraphPlugin
           canEditDrillDownUrls: config.canEditDrillDownUrls,
           graphSavePolicy: config.savePolicy,
           storage: new Storage(window.localStorage),
-          capabilities: coreStart.application.capabilities.graph,
+          capabilities: coreStart.application.capabilities,
```

Comment:
While reviewing the code I noticed, that in this file 

`pluginsStart.kibanaLegacy.loadAngularBootstrap()`

is used, think this is no longer necessary? BTW since there is no more Angular inside (this is awesome 🥳 ), there's still some angular naming and labeling under the hood which could be cleaned up (but this could also be a follow up PR)

Reply:
This line removed, thank you.